townhall/myapi/tests/test_organization.py

```diff
@@ -6,3 +6,68 @@
 
 # Testing ALL tests "python manage.py test myapi.tests"
 # Testing ONLY organization "python manage.py test myapi.tests.test_organization"
+
+class TestOrganizationModel(TestCase):
+    def setUp(self):
+        townhall_models.Organization.objects.create(id=1, name="Goodwill", location="Victoria", email="goodwill@gmail.com", phone_number="778-123-4567", website="goodwill.ca")
+        townhall_models.Organization.objects.create(id=2, name="Salvation Army", location="Richmond", email="salvationarmy@hotmail.ca", phone_number="604-987-6543", website="salvationarmy.com")
+
+    def test_get_organization(self):
+        organization = townhall_services.OrganizationServices.get_organization(id=1)
+        assert organization.name == "Goodwill"
+        assert organization.location == "Victoria"
```

Comment:
There's a lot of[ magic strings](https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings) in this test file which is a slight problem, the way to fix it (and we can do this later not in this PR) is to either use fixtures/mocks or dataclasses.